src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -25,13 +26,29 @@ static const wchar_t *serverGcVar = W("COMPlus_gcServer");
 // On by default.
 static const wchar_t *concurrentGcVar = W("COMPlus_gcConcurrent");
 
+// Environment variable for setting whether to swallow uncaught exceptions (for testing)
+static const wchar_t *swallowUncaughtExceptionsVar = W("COMPlus_Corerun_Swallow_Uncaught_Exceptions");
+static const wchar_t *useEntrypointFilterVar = W("COMPlus_Corerun_Use_EntryPoint_Filter");
+
```

Comment:
There is a separate corerun for Unix. Do we need to test what this does on Unix too?

Reply:
I'm not sure how to test on Linux.  @vitek-karas had suggested enabling this only for Windows since this was particularly for IIS.  I think I'll head that direction.

Reply:
This is now disabled except on Windows